package.json

```diff
@@ -20,7 +20,10 @@
   "author": "Gnosis",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.17.1"
+    "@gnosis.pm/dex-contracts": "file:.yalc/@gnosis.pm/dex-contracts",
```

Comment:
This is cool - is this for testing a package without actually publishing?

Reply:
yes!